web/src/lib/components/album-page/album-card.svelte

```diff
@@ -83,14 +83,25 @@
   {/if}
 
   <div class={`relative aspect-square`}>
-    <img
-      loading={preload ? 'eager' : 'lazy'}
-      src={imageData}
-      alt={album.id}
-      class={`z-0 h-full w-full rounded-xl object-cover transition-all duration-300 hover:shadow-lg`}
-      data-testid="album-image"
-      draggable="false"
-    />
+    {#if album.albumThumbnailAssetId}
+      <img
+        loading={preload ? 'eager' : 'lazy'}
+        src={imageData}
+        alt={album.id}
+        class={`z-0 h-full w-full rounded-xl object-cover transition-all duration-300 hover:shadow-lg`}
+        data-testid="album-image"
+        draggable="false"
+      />
+    {:else}
+      <enhanced:img
+        loading={preload ? 'eager' : 'lazy'}
+        src={noThumbnailUrl}
```

Comment:
You should be able to do this and then remove the `import` above:
```suggestion
        src="@assets/no-thumbnail.png"
```

Reply:
Although maybe it's be better to leave the import and then do something like `<enhanced:img src={imageData || noThumbnailUrl} />` where you have only a single image. I'm not 100% sure that would work, but I think it should. I might be able to put together a fix for it if it doesn't work

Reply:
It looks pixelated unless I provide the dimensions with `?w=271;186`, do you know a better way ?

Reply:
Hmm. Including the dimensions should have no effect since those are the original image dimensions. Do you need to do that when referring to it directly via the `src` attribute, when importing it, or both?

Reply:
Providing the dimensions via the `src` or `sizes` attributes fix it :
- `sizes="min(271px,186px)"`

or

- `src="$lib/assets/no-thumbnail.png?w=271;186"`

(271x186 are the dimensions of the original picture)